library/dash/src/main/java/com/google/android/exoplayer2/source/dash/manifest/AdaptationSet.java

```diff
@@ -65,19 +71,22 @@
    * @param accessibilityDescriptors Accessibility descriptors in the adaptation set.
    * @param essentialProperties Essential properties in the adaptation set.
    * @param supplementalProperties Supplemental properties in the adaptation set.
+   * @param producerReferenceTimeInfo Producer Reference Time Info
    */
   public AdaptationSet(
-      int id,
-      int type,
-      List<Representation> representations,
-      List<Descriptor> accessibilityDescriptors,
-      List<Descriptor> essentialProperties,
-      List<Descriptor> supplementalProperties) {
+          int id,
```

Comment:
unexpected identation change

Reply:
Another param is added here: `producerReferenceTimeInfo`